engine/api/src/main/scala/pl/touk/nussknacker/engine/api/process/ExpressionConfig.scala

```diff
@@ -14,7 +14,8 @@ case class ExpressionConfig(globalProcessVariables: Map[String, WithCategories[A
                             //      DictInstance have the same dictId as DictDefinition
                             dictionaries: Map[String, WithCategories[DictDefinition]] = Map.empty,
                             hideMetaVariable: Boolean = false,
-                            strictMethodsChecking: Boolean = true
+                            strictMethodsChecking: Boolean = true,
+                            referenceTypeValidating: Boolean = false
```

Comment:
I would prefer single  flag for both type refernce and theirs method checking. Also WDYT about having "static" in the name? Now this name is very spel implementation details specific. From user perspecitve it is something like just "staticMethodInvocationsChecking"

Reply:
Changed the flag name, thanks